pkg/restapi/v1/verifier/models.go

```diff
@@ -0,0 +1,14 @@
+/*
+Copyright SecureKey Technologies Inc. All Rights Reserved.
+
+SPDX-License-Identifier: Apache-2.0
+*/
+
+package verifier
+
+import "github.com/trustbloc/vcs/pkg/service/verifycredential"
+
+// CredentialsVerificationResponse represents the response of credentials verification.
+type CredentialsVerificationResponse struct {
```

Comment:
I think this should be generated from openapi.yaml. You can define there this structure

Reply:
added